doc/windows/package-manager/package/repository.md

```diff
@@ -60,28 +62,29 @@ Next, create a fork of the repository and clone it.
 
 You must add your manifest file to the repository in the following folder structure:
 
-**manifests** / **publisher** / **application** / **version.yaml**
+**manifests** / **letter** / **publisher** / **application** / **version** / **Yaml file**
 
 * The **manifests** folder is the root folder for all manifests in the repository.
+* The **letter** folder is the first letter of the publisher name.
 * The **publisher** folder is the name of the company that publishes the software. For example, **Microsoft**.
 * The **application** folder is the name of the application or tool. For example, **VSCode**.
-* **version.yaml** is the file name of the manifest. The file name must be set to the current version of the application. For example, **1.0.0.yaml**.
+* The **version** folder is the version of the application or tool. For example, **1.0.0**.
+* **Yaml File** is the file name of the manifest. The file name must be set to the name and publisher of the application. For example, **Contoso.ContosoApp.yaml**.
 
->[!IMPORTANT]
-> The `PackageIdentifier` value in the manifest must match the publisher and application names in the manifest folder path, and the `PackageVersion` value in the manifest must match the version in the file name. For more information, see [Create your package manifest](manifest.md#tips-and-best-practices).
+The `PackageIdentifier` value in the manifest must match the publisher and application names in the manifest folder path, and the `PackageVersion` value in the manifest must match the version in the file name. For more information, see [Create your package manifest](manifest.md#tips-and-best-practices).
 
 ### Step 4: Submit your manifest to the remote repository
 
 You're now ready to push your new manifest to the remote repository.
 
 1. Use the `add` command to prepare for submission.
     ```CMD
-    git add manifests\Contoso\ContosoApp\1.0.0.yaml
+    git add manifests\C\Contoso\ContosoApp\1.0.0\Contoso.ContosoApp.yaml
     ```
 
 2. Use the `commit` command to commit the change and provide information on the submission.
     ```CMD
-    git commit -m "Submitting  ContosoApp version 1.0.0.yaml"
+    git commit -m "Submitting ContosoApp version 1.0.0.yaml"
```

Comment:
This should probably be "Submitting ContosoApp version 1.0.0." since we don't submit one file without an enclosing folder anymore.